backend/app/services/users/put_avatar.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Users
+  class PutAvatar < Core::Service
+    def call(url_params, current_user)
```

Comment:
This service is supposed to attach an avatar to the **provided** user. (I.e. we don't care about `current_user`, only for provided `user`